frontend/src/modules/profile/preview.jsx

```diff
@@ -77,7 +77,8 @@ const findCountries = (countries, item) => {
 };
 
 export const GeneralPreview = ({ item }) => {
-  const { countries, tags, meaOptions } = UIStore.currentState;
+  const globalState = UIStore.useState();
```

Comment:
here we re-rendered each time every piece of data of UIStore changes ... 
https://lostpebble.github.io/pullstate/docs/use-store-state-hook#use-a-stores-entire-state vs https://lostpebble.github.io/pullstate/docs/use-store-state-hook#use-a-sub-state-of-a-store

Reply:
do we really need this behaviour here?